Sources/ArcGISToolkit/Common/CodeScanner.swift

```diff
@@ -263,14 +273,46 @@ class ScannerViewController: UIViewController, @preconcurrency AVCaptureMetadata
         return path
     }
     
+    /// Checks if the reticle intersects with any of the current overlays. When a code with a consistent string
+    /// value intersects with the reticle for the `requiredTargetHits` count, it is auto-scanned.
+    private func checkTargetHits() {
+        var reticleWasContainedInAOverlay = false
```

Comment:
Just a little grammar change. Here and in the other places it's used.
```suggestion
        var reticleWasContainedInAnOverlay = false
```